geometry_msgs/Pose pose
bool grasp